### PR TITLE
INT-238 Migrate Partner API docs from Doku Wiki to Github Pages

### PR DESCRIPTION
**Pull request by hx-markterry**
https://holidayextras.jira.com/browse/INT-238

#### What does this PR do? (please provide any background)

This work takes an old dump of the doku wiki pages and converts them to markdown for github.

#### What tests does this PR have?

None.

#### How can this be tested?

A selection of the files above will need to be proof read to ensure that the doku wiki formatting is correctly converted.

    git clone (repo)
    cd partner-api-docs
    git checkout INT-238
    bundle install
    bundle exec jekyll serve

(Wait 2 mins)

    open http://localhost:4000

#### Any tech debt?

The conversion might not be 100% but there could be diminishing returns in getting a perfect automatic conversion.

#### What gif best describes how you feel about this work?

giphy

I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

By approving a review you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---

### CODE REVIEW

**Comment by hpoom** [comment]
👍

File: pages/de_hxapi/e-mail/e-mail_notification_usage.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+{{e-mail:systems_evolution_screenshot.png|}}
+
+
+
+
+
+## Usage Details
+
+There is currently no interface for creating new e-mail notifications. Contact [Philip Carrington](philip.carrington@holidayextras.com) if you need a new report.
```
Comment (PhilCarr) [inline pages/de_hxapi/e-mail/e-mail_notification_usage.md]: Please can we change this to read something like:
There is currently no interface for creating new e-mail notifications. Please contact your Account Manager and they can arrange report creation.

File: pages/e-mail/e-mail_notification_usage.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+{{e-mail:systems_evolution_screenshot.png|}}
+
+
+
+
+
+## Usage Details
+
+There is currently no interface for creating new e-mail notifications. Contact [Philip Carrington](philip.carrington@holidayextras.com) if you need a new report.
```
Comment (PhilCarr) [inline pages/e-mail/e-mail_notification_usage.md]: Please can we change this to read something like:
There is currently no interface for creating new e-mail notifications. Please contact your Account Manager and they can arrange report creation.

**Comment by carolinewoodhams** [comment]
**Landing page:**
- both 'Holiday Extras API' and 'Official Documentation' are H1 headings (this happens on all linked out pages too)

**Overview:**
- link to 'XML feed' takes to non-docs link (not necessarily part of this PR, but think we should look at whether we move this content over as well in the future).
- 'what's in a request' - broken link to RESTful API info

**System requirements:**
- 'how to cancel a booking' - 'faq' links to old doku wiki site
- 'specific documentation' - 'holiday extras endpoints' takes you through to index of /hxapi/, which is not formatted as rest of docs

**Endpoints:**
- Tables, such as those used in 'barcode' -> 'parameters', are not formatting correctly.
- 'Barcode' - A link at the bottom of page looks to be broken (
http://chart.apis.google.com/chart?cht=qr&chs=300x300&chl=HX123&chld=H0)
- 'carhire.html' is broken - index appears unformatted (actual pages are ok) - same for de.parking, 
- 'de-start-index' - all the links are broken (and the table is unformatted).
- 'exampleapps' - pointless?

Have got about half way through endpoints - need to go home but can pick up tomorrow again (hope this is enough feedback so far to work with :)

**Comment by hx-markterry** [comment]
@carolinewoodhams I've created issues for the items suggested above that we can fix as other work.

**Comment by carolinewoodhams** [comment]
I've reviewed all the endpoints (following the links all the way through to make sure it's all working ok). 

The only thing I could come across as an issue is the _View / amend / cancel_ - it feels like there is something missing from this page. It lacks content, whereas I think previously it had more to say on how to action these requests.

However, I think that's outside the scope of this ticket now. All other issues I previously raised have been fixed. 

I LOVE that we have done this, I am itching to get on and change the content and make it really super fab. Great first step.

**Pull request merged**